diff --git a/contracts/RecapitalizationModule.sol b/contracts/RecapitalizationModule.sol
index d26dcd9..1dd682c 100644
--- a/contracts/RecapitalizationModule.sol
+++ b/contracts/RecapitalizationModule.sol
@@ -46,6 +46,22 @@ contract RecapitalizationModule is IRecapitalizationModule {
         maximumIssuanceRate = maximumIssuanceRate_;
     }
 
+    /**************************************************************************************************************************************/
+    /*** Certora Helper Functions                                                                                                       ***/
+    /**************************************************************************************************************************************/
+
+    function getNextWindowId(uint16 windowId) external view returns (uint256 nextWindowId_) {
+        nextWindowId_ = windows[windowId].nextWindowId;
+    }
+
+    function getWindowStart(uint16 windowId) external view returns (uint256 windowStart_) {
+        windowStart_ = windows[windowId].windowStart;
+    }
+
+    function getIssuanceRate(uint16 windowId) external view returns (uint256 issuanceRate_) {
+        issuanceRate_ = windows[windowId].issuanceRate;
+    }
+
     /**************************************************************************************************************************************/
     /*** External Functions                                                                                                             ***/
     /**************************************************************************************************************************************/
